aioredis/connection.py

```diff
@@ -78,7 +82,12 @@
 
 SERVER_CLOSED_CONNECTION_ERROR = "Connection closed by server."
 
-SENTINEL = object()
+
+class Sentinel(enum.Enum):
+    sentinel = object()
```

Comment:
Why is this `enum.Enum` needed? And can you prepend an underscore to `Sentinel`?

Reply:
Sentinels are a little messy to type. It seems that an enum is the best approach to use:
https://stackoverflow.com/a/60605919